test/image/mocks/animation_bar.json

```diff
@@ -0,0 +1,22 @@
+{
+  "data": [{
+    "type": "bar",
+    "x": ["A", "B", "C"],
+    "y": [24, 5, 8],
+    "error_y": {"array": [3, 2, 1]}
+    }],
+  "layout": {
+    "width": 400,
+    "height": 400,
+    "yaxis": {"range": [0, 30]}
+  },
+  "frames": [
```

Comment:
For reference, calling `Plotly.animate(gd)` off this mock is very fun:


![Peek 2019-09-10 11-50](https://user-images.githubusercontent.com/6675409/64629466-3f8f1000-d3c1-11e9-8d68-c33f7508785f.gif)